src/partials/logo.html

```diff
@@ -0,0 +1 @@
+<h1 class="logo">english<span class="accent">excellence</span>.</h1>
```

Comment:
cant use H1 tag for a logo

Reply:
Removed logo partial completely, renamed CSS class